dbt/models/staging/stg_candidatures.sql

```diff
@@ -22,10 +22,7 @@ select
             then 'Motif "Autre" saisi sur les emplois'
         else candidatures.motif_de_refus
     end                                                    as motif_de_refus,
-    case
```

Comment:
Faudrait qu'on fasse ça plus proprement et qu'on récupère le truc invariable des emplois...et ensuite un seed et clic clac kodak.

Je suis pas fan de faire les choses quick and dirty surtout pour un truc qui m'a pas l'air urgent